app/models/index.js

```diff
@@ -1,35 +1,36 @@
-const dbConfig = require("../config/db.config.js");
+const Sequelize = require('sequelize');
+const dbConfig = require('../config/db.config.js');
 
-const Sequelize = require("sequelize");
 const sequelize = new Sequelize(dbConfig.DB, dbConfig.USER, dbConfig.PASSWORD, {
   host: dbConfig.HOST,
   dialect: dbConfig.dialect,
   operatorsAliases: false,
 
   define: {
-    timestamps: false
+    timestamps: false,
   },
 
   pool: {
     max: dbConfig.pool.max,
     min: dbConfig.pool.min,
     acquire: dbConfig.pool.acquire,
-    idle: dbConfig.pool.idle
-  }
+    idle: dbConfig.pool.idle,
+  },
 });
 
 const db = {};
 
-Object.keys(db).forEach(modelName => {
```

Comment:
This was in the wrong order and was doing nothing because it was running on an empty object from line 21. It has to be after the assignment of the different models to the db object in order to work.